frontend/src/app/services/question/question.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Question } from '@app/model/Question';
+import { environment } from '@environments/environment';
+import { first, map } from 'rxjs/operators';
+import { parseQuestion } from '@app/parsers/question.parser';
+import { YesNoQuestion } from '@app/model/PatientInfo';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class QuestionService {
+
+  constructor(private _http: HttpClient) {
+  }
+
+  public async getQuestions(): Promise<YesNoQuestion[]> {
+    return this._http.get<Question[]>(
+      `${environment.apiUrl}/question`
```

Comment:
Nejsem si jisty, ale nebylo to `questions`? Ale mozna to pak Lukas zmenil

Reply:
podle swaggeru `question`

Reply:
![image](https://user-images.githubusercontent.com/14038818/109935513-f7a72b80-7ccd-11eb-8555-aaad44c92f1e.png)
a swagger nikdy nelze